basket/src/components/Login.tsx

```diff
@@ -0,0 +1,19 @@
+type LoginProps = {
+  login: (id: number, name: string) => void;
+  inputValue: string;
+  setInputValue: React.Dispatch<React.SetStateAction<string>>;
+};
```

Comment:
이 부분은 이렇게 타입을 작성할 수도 있는데 취향이라 팀에서 어떻게 사용하고 있냐에 따라서 팀에 맞추기!

```suggestion
import { Dispatch, SetStateAction } from 'react';

type LoginProps = {
  login: (id: number, name: string) => void;
  inputValue: string;
  setInputValue: Dispatch<SetStateAction<string>>;
};
```

Reply:
이 부분에 대해서는 회사 코드를 보고 어떻게 쓸지 결정해보겠습니닷!